Release v7.1.0 (#1713)
kedro-org/kedro-viz: RELEASE.md

- Include the ID number for the related PR (or PRs) in parentheses
-->

# Release 7.1.0

## Major features and improvements

> - Add build command to the CLI using `kedro viz build` for creating build directory of local Kedro Viz instance with Kedro project data. (#1697)

Reviewer: ```suggestion
- Add build command `kedro viz build` to the CLI to allow users to create a build directory of local Kedro Viz instance with Kedro project data. (#1697)
```
